src/Components/LoginForm.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react'
+import { Form, Button, Container, Message } from 'semantic-ui-react'
+import { connect } from 'react-redux'
+import { signInUser } from '../reduxTokenAuthConfig'
+
+class LoginForm extends Component {
+  state = {
+    email: '',
+    password: '',
+    message: '',
+    errorsLogin: ''
+  }
+
+  onChangeHandler = (e) => {
+    this.setState({
+      [e.target.id]: e.target.value
+    })
+  }
+
+  onSubmit = (e) => {
+    e.preventDefault();
+    const { history, signInUser } = this.props
+    const {
+      email,
+      password
+    } = this.state
+    signInUser({ email, password })
+      .then(response => {
+        this.setState({ message: true })
+        setTimeout(function () { history.push('/') }, 3000)
+      }).catch(error => {
+        this.setState({
+          errorsLogin: error.response.data.errors[0],
+          message: false
+        })
+      })
+  }
+
+  render() {
+
+    let message
+
+    if (this.state.message === true) {
```

Comment:
refactor this to check the redux store if user is `isSignedIn`

Reply:
done